docs/src/3.1.Worldview.mdx

```diff
@@ -11,6 +11,7 @@
 | `shiftKeys`           | `boolean`                                       | true                   | enable/disable shift modifier for keyboard controls                                                     |
 | `backgroundColor`     | `Vec4`                                          | `[0,0,0,1]`            | change the scene background color                                                                       |
 | `hitmapOnMouseMove`   | `boolean`                                       |                        | if enabled, the clicked object will be returned from the mouse event handler. Disabled by default for performance reasons. |
```

Comment:
This one seems now redundant with `disableHitmapForEvents`?

Reply:
Ah I see you marked it deprecated below — should we just remove it from the docs then?

Reply:
I prefer to leave it for now, since the property is deprecated and not really removed. We should remove it on a future major release, though.

Reply:
Then at least let's update the docs to make very clear that this is deprecated and that moving forward people should use `disableHitmapForEvents` instead. Maybe move it to the bottom of the table, too?